src/parser/jobs/whm/Benison.js

```diff
@@ -0,0 +1,81 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+
+const WASTED_USES_MAX_MINOR = 3
+const WASTED_USES_MAX_MEDIUM = 10
+
+export default class Benison extends Module {
+	static handle = 'benison'
+	static dependencies = [
+		'checklist',
```

Comment:
You're not using most of these dependencies. Avoid depending on things not in use, it just complicates things a little.